test/Interactive/Tests.hs

```diff
@@ -13,10 +13,24 @@ import System.Exit
 
 import Agda.Utils.Monad
 import Utils
+import Test.Tasty.Silver.Filter (RegexFilter (RFInclude))
 
 testDir :: FilePath
 testDir = "test" </> "Interactive"
 
+-- | Filtering out interaction tests that require Agda built with -fdebug.
+
+fdebugTestFilter :: [RegexFilter]
+fdebugTestFilter =
+-- This list was crafted using
+--    grep -RP '(?<!-- ){-# OPTIONS.* -v' | grep interaction/
+--  and screening the results (e.g. for comments)
+  [ disable "interaction/Debug"
+  , disable "interaction/Issue1353"
+  , disable "interaction/Positivity"
+  ]
```

Comment:
There is a mixup between `Interactive` (tests the decommissioned Agda repl) and `interaction` (tests the Agda interaction facilities).
The `interaction` tests are still run by a `Makefile` only, do not have a implementation as `tasty-silver` testsuite.
So I suppose this code is not doing anything here.
I'll proceed to remove it.
It would have to be ported to the `Makefile`, but as I understood, we are anyway not testing this PR in CI.

Reply:
Removed in #7604.